enterprise/frontend/src/embedding-sdk-bundle/components/public/SdkQuestion/SdkQuestion.unit.spec.tsx

```diff
@@ -202,13 +193,7 @@ describe("InteractiveQuestion", () => {
   });
 
   it("should render when question is valid", async () => {
-    setup();
-
-    await waitForLoaderToBeRemoved();
-
-    act(() => {
-      jest.runAllTimers();
-    });
```

Comment:
I don't believe jest timers weren't necessary in the tests, the console was showing warnings about them and the tests work fine without them